### PR TITLE
feat: adding initial version of the static website DA

### PR DESCRIPTION
**Pull request by sendooranS**
### Description

These changes add in the revised, production code for the static website DA. The only difference between the previous reviewed production code and this is the change to the url of the diagram.

#### Changes that affect the core Terraform module or submodules
- [ ] Bug fix
- [x] New feature
- [ ] Dependency update

#### Changes that don't affect the core Terraform module or submodules
- [ ] Examples or tests (addition or updates of examples or tests)
- [ ] Documentation update
- [ ] CI-related update (pipeline, etc.)
- [ ] Other

#### Release required?
Identify the type of release. For information about the changes in a semantic versioning release, see [Release versioning](https://terraform-ibm-modules.github.io/documentation/#/versioning).

- [ ] No release
- [ ] Patch release (`x.x.X`)
- [x] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

##### Release notes content

- Introduces the logic for the static website deployable architecture

---

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### Merge actions for mergers

- Use a relevant [conventional commit](https://www.conventionalcommits.org/) message that is based on the PR contents and any release notes provided by the PR author. The commit message determines whether a new version of the module is needed, and if so, which semver increment to use (major, minor, or patch).
- Merge by using "Squash and merge".

### CODE REVIEW

File: cos/scripts/cos.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env sh
+####################
+set -e
+
+IAM_TOKEN=$(curl -X POST "${IBMCLOUD_IAM_ENDPOINT}" \
+    -H 'Accept: application/json' \
+    -H 'Content-Type: application/x-www-form-urlencoded' \
+    --data-urlencode 'grant_type=urn:ibm:params:oauth:grant-type:apikey' \
+    --data-urlencode "apikey=${APIKEY}" | jq -r .access_token)
```
Comment (nkseto) [inline cos/scripts/cos.sh:9]: We should update the README to indicate that jq is a requirement to run this

File: cos/html/index.html [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
```
Comment (nkseto) [inline cos/html/index.html:1]: Where are the files under cos/html being referenced?  I don't see it being consumed so I'll suggest this folder to be removed.

File: cos/scripts/cos.sh (hunk 1 above)
Comment (sendooranS) [reply cos/scripts/cos.sh:9]: handled in https://github.com/terraform-ibm-modules/terraform-ibm-static-website/pull/2/commits/a181a56efd65d0daf35c34186841de25bddae7b6

File: cos/html/index.html (hunk 2 above)
Comment (sendooranS) [reply cos/html/index.html:1]: I believe we store the index.html in the cos instance: https://github.com/terraform-ibm-modules/terraform-ibm-static-website/blob/initial-version-code-dump/cos/main.tf#L43

**Review by nkseto — APPROVED** [review]

**Pull request merged**